### DISCUSSION TITLE
Any chance of adding to the Homebrew Libraries?

### DISCUSSION THREAD

**Discussion by BobHood** [Alex313031/Thorium-MacOS#13] [Ideas]
Tried to install with Homebrew (thinking that it was there), and some book reading software was in there. Any chance of adding it to the Homebrew libraries (as thorium-browser)?

**Comment by gz83** [comment]
Thanks for the advice.

@BobHood

**Comment by earthsound** [comment]
For anyone else who is looking, it's available.

Run:
`brew install alex313031-thorium`
